### ISSUE TITLE
Blog Post: Internal Airflow Use Case (Clickstream)

### ISSUE THREAD

**Issue by paolaperaza** [astronomer/airflow-guides#46]
Originally @vparekh94's idea. 

We don't have any public documentation about our internal Airflow use between Clickstream and Internal reporting.

@rhwang10 and I will take the lead on getting this started, based on [this](https://github.com/astronomerio/spacecamp/blob/master/modules/1_data_engineering_and_airflow/6_case_study_internal_use/Airflow%20at%20Astronomer.ipynb
) SpaceCamp module.

**Comment by paolaperaza** [comment]
@rhwang10 and I met today to get started on this. We started off nailing down what we're thinking for general structure. Outline found here: https://docs.google.com/document/d/1aR2ZMamOP5UJSFee7eFSCk95Zj0fy1E-9YEodkpA9Aw/edit

**Next steps:** I’m gonna draft parts I and II, and he’s gonna draft the second half.

Meeting set in the books for Thursday mid morning, and we’re hoping to have a final draft on Friday for people to review and publish on Monday.

**Comment by tedmiston** [comment]
Lmk if you need any technical deets from me on this around the Clickstream DAGs, Spark Streaming, etc.

If this post is getting into technical details, the migration I'm currently doing is getting the whole system running on AWS EMR for Vortex and Astronomer EE on AWS EKS for the Clickstream Redshift Loader.  That brings a massive improvement over the legacy cloud setup on DC/OS.

**Comment by vparekh94** [comment]
@paolaperaza @rhwang10 how is this looking?

**Comment by rhwang10** [comment]
It's ready to publish, but Tim wanted us to hold it until metarouter has a site up

**Comment by paolaperaza** [comment]
^ +1

I actually checked in with Tim this morning, and he said "Hey Paola - hoping for next week. Need to get a basic website built first."

**Comment by vparekh94** [comment]
@paolaperaza are we waiting on metarouter for this one? If so, we should close this out and open another issue in the relevant repo

**Comment by paolaperaza** [comment]
lol aaaaaand finally closing this.

Blog post live here: https://www.astronomer.io/blog/astronomer-on-astronomer-internal-use-case/
